composable.yml

```diff
@@ -99,31 +99,7 @@
       args:
         chdir: "/home/stack"
 
-    - name: get node UUIDs
-      shell: |
-        source ~/stackrc
-        openstack baremetal node list --format value -c UUID
-      register: node_uuid
-
-    - name: get the node fqdn list
-      shell: |
-        source ~/stackrc
-        openstack baremetal node show {{ item }} --fields driver_info -f json | jq '.driver_info.ipmi_address'
-      register: node_fqdn
-      loop: "{{ node_uuid.stdout_lines|flatten(levels=1) }}"
-
-    - name: populate uuid type map
-      set_fact:
-        uuid_type_map: "{{ uuid_type_map|default({}) | combine ( {item.0 : type}) }}"
+    - name: setting node capabilities (boot_mode bios)
       vars:
-        type1: "{{ (lab_name == 'scale') | ternary(item.1.stdout.split('.')[0].split('-')[4], item.1.stdout.split('.')[0].split('-')[3]) }}"
-        type: "{{ (type1 != '1029u') | ternary(type1, (item.1.stdout.replace('mgmt-', '').replace('\"', '') in tn10rt) | ternary('1029utn10rt', '1029utrtp')) }}"
-      with_together:
-        - "{{ node_uuid.stdout_lines }}"
-        - "{{ node_fqdn.results }}"
-
-    - name: setting profile for node type
-      shell: |
-        source ~/stackrc
-        openstack baremetal node set {{ item.key }} --property capabilities=profile:baremetal{{ item.value }},cpu_vt:true,cpu_hugepages:true,boot_option:local,cpu_txt:true,cpu_aes:true,cpu_hugepages_1g:true,boot_mode:bios
-      loop: "{{ uuid_type_map|dict2items }}"
+        boot_mode: bios
```

Comment:
The ironic team mentioned that OSP 17 is leaning towards using UEFI boot mode as default. I think we might want to work with the lab team to check why UEFI doesn't work usually for supermicro hosts, so that we can use UEFI instead of BIOS in the future.

Reply:
> I think the neutron team has plans to make RAFT clusters the default for OVN in the future, so Jetpack should also support it. Passing these parameters to the infrared deploy command enables OVN RAFT. `--config-resource OS::TripleO::Services::OVNDBs=/usr/share/openstack-tripleo-heat-templates/deployment/ovn/ovn-dbs-cluster-ansible.yaml`

This is already supported in jetpack